File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        best, res = 0, 0
        for i in range(len(nums)):
            t = nums[i]
            if not t:
                best = max(best, res)
                res = 0
                continue
            res = res + t
        return max(best, res)

File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> sums = new HashMap<>();
        int[] res = new int[2];

        for(int i = 0; i < nums.length; i++) {
            int value = nums[i];
            int tmp = target - value;

            if(!sums.containsKey(tmp)) {
                sums.put(value, i);
            } else {
                res[0] = sums.get(tmp);
                res[1] = i;
                return res;
            }
        }
        return res;
    }
}
